zingolib/src/wallet/notes/sapling.rs

```diff
@@ -189,104 +204,37 @@ impl ShieldedNoteInterface for SaplingNote {
 #[cfg(any(test, feature = "test-features"))]
 pub mod mocks {
```

Comment:
if this mod is empty, lets remove it

Reply:
I think we'll want to reorganize mods a bit, perhaps as it's own PR (including removing this empty one).   I thought it should be it's own mod/name PR.

Reply:
ok, open more PRs for these good ideas